modules/test/playwright/tests/object-web/main/objectFields.spec.ts

```diff
@@ -1216,6 +1216,102 @@ test.describe('Manage objectFields through Objects Admin UI', () => {
 		).toBeVisible();
 	});
 
+	test('cannot delete system fields of system objects', async ({
+		apiHelpers,
+		objectFieldsPage,
+		page,
+	}) => {
+		await objectFieldsPage.goto('User');
+
+		const objectDefinitionAPIClient =
+			await apiHelpers.buildRestClient(ObjectDefinitionAPI);
+
+		const {body: objectDefinition} =
+			await objectDefinitionAPIClient.getObjectDefinitionByExternalReferenceCode(
+				'L_USER'
+			);
+
+		const systemFields = objectDefinition.objectFields.filter((item) => {
+			if (item.system) {
+				return item;
+			}
+		});
+
+		for (const item of systemFields) {
+			await expect(
+				page
+					.getByRole('row', {name: item.label.en_US})
+					.getByRole('button', {name: 'Actions'})
+			).toBeHidden();
+		}
+	});
+
+	test('cannot delete a custom object relationship field from a system object', async ({
```

Comment:
hey @DavysonMelo, could you sort this test? I believe this one should be before 
`test('cannot delete an objectField that belongs to a unique composite key validation through Objects Admin UI'`